lib/index.js

```diff
@@ -16,14 +16,10 @@ injector.use('jsonld-signatures', jsigs);
 
 injector.env = {nodejs: true};
 
-const VeresOneClient = require('./methods/veres-one-client.js');
+const WebLedgerClient = require('./web-ledger-client.js');
 
 module.exports = {
   injector,
   use: (name, injectable) => injector.use(name, injectable),
-  methods: {
-    veres: (options) => {
-      return new VeresOneClient({injector, ...options});
-    }
-  }
+  client: WebLedgerClient
```

Comment:
Do you think it might make sense to expose `WebLedgerClient` directly, not via `client`? 
Cause right now my downstream import (of this branch) looks like:

```
const WebLedgerClient = require('web-ledger-client').client;
```

Reply:
Yes, that makes more sense. I was trying to make the minimum amount of code changes in order to gain the functionality that I needed for rapid prototyping. There will need to be a second pass of changes that consists of refactoring this library so that the API is more sensible and intuitive.

Reply:
@gannan08 So, I'm thinking that we have one of two choices.

Either we return a bare `WebLedgerClient`, like so:

```js
module.exports = WebLedgerClient;
```
In which case the consuming code will be:

```js
const WebLedgerClient = require('web-ledger-client');

const client = new WebLedgerClient({ injector, etc... }); // pass in your own existing injector
```

OR we expose a factory method, like:

```js
module.exports = {
  injector,
  use: (name, injectable) => injector.use(name, injectable),
  client: (options) => new WebLedgerClient({injector, ...options});
};
```
in which case the consuming code would be something like:

```js
const wlc = require('web-ledger-client');

const client = wlc.client({ ... }); // new client instance
```

I'm totally fine with either one (and the first option might be easiest, since our consuming code is very likely to have its own injector instance anyway).

Reply:
I vote for returning a bare `WebLedgerClient`.